bin/oscapd-evaluate

```diff
@@ -566,8 +566,10 @@ def main():
     config.prepare_dirs(cleanup_allowed=False)
     try:
         config.sanity_check()
-    except:
-        logging.exception("Configuration file failed sanity checking!")
+    except Exception as exc:
```

Comment:
Can it catch a specific class of exception instead?

Reply:
The sanity checking code is throwing `RuntimeError`, but I think that we want to catch all exceptions and report back that they occurred during config file sanity checking.